app/app.test.js

```diff
@@ -18,58 +18,52 @@ const expectedPages = [
   '/examples/template-custom'
 ]
 
-// Returns a wrapper for `request` which applies these options by default
-const requestPath = request.defaults({
-  baseUrl: `http://localhost:${PORT}`,
-  headers: {
-    'Content-Type': 'text/plain'
-  }
-})
+// Reduce test keep-alive
+setGlobalDispatcher(new Agent({
```

Comment:
@36degrees Thanks for the link on this bit, hopefully keeps the test runs short and sweet

Ready to review again now